src/components/SharedComponents/ActivityCount/ActivityCount.js

```diff
@@ -13,18 +14,19 @@ type Props = {
   color: string,
   icon?: string,
   accessibilityLabel: string,
-  testID?: string
+  testID?: string,
+  marginClass?: string
 }
 
 const ActivityCount = ( {
-  count, color, icon, accessibilityLabel, testID
+  count, color, icon, accessibilityLabel, testID, marginClass = ""
```

Comment:
Any reason to use a blank string over null here?

Reply:
No reason other than keeping the types the same. `classnames` ignores falsey values.

Reply:
Fair enough.